logbook-core/src/main/java/org/zalando/logbook/CookieHeaderFilter.java

```diff
@@ -0,0 +1,102 @@
+package org.zalando.logbook;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Predicate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.compile;
+import static java.util.stream.Collectors.toList;
+
+final class CookieHeaderFilter implements HeaderFilter {
+
+    private final Pattern pattern = compile("(?<name>[^=; ]+)=(?<value>[^; ]*)");
```

Comment:
`sessionToken=` would be replaced by `sessionToken=XXX` - is that intended? I guess that's fine as hiding even the empty string may be favorable.

Reply:
That's worth adding a test case

Reply:
Done